docs/building-images.md

```diff
@@ -40,6 +40,8 @@ operatingSystem:
   isoConfiguration:
     installDevice: /path/to/disk
     unattended: false
+  rawConfiguration:
```

Comment:
This was initially intentionally omitted due to the conflict with `isoConfiguration`. Not sure if we want to keep it.

Reply:
I think we should have it in the examples and maybe specify in the parameter options that it should not be used in cohesion with the `isoConfiguration` parameter. Otherwise users might miss it (as did I). Nevertheless, if you think that this is not a good idea I do not mind removing it.

Reply:
Iterating over this, actually we do not need an example, as we ensure that the user will know about the `rawConfiguration ` property from the note in the `Kubernetes` section. Property example has been removed.